benchmarks/htap/lib/monitoring.py

```diff
@@ -181,14 +185,28 @@ def db_size(self):
         return "{:7.2f}GB".format(self.cached_database_size / (1024*1024*1024.0))
 
     def columnstore_stats(self):
-        return self.cached_columnstore_stats
+        result = []
+        for row in self.cached_columnstore_stats:
+            table_name = row['table_name']
+            table_size = row['relation_blocks']*8/1024/1024;
+            index_size = row['compressed_blocks']*8/1024/1024;
+            compression_ratio = table_size * 1.0 / index_size
```

Comment:
Don't we need some parenthesis here to have explicit precedence between * and / ?
also for the next line.

Reply:
no we don't. can add them if you like but the 1.0 is only there to convert the datatype.

Reply:
ok no problem then